server/routes/varyingLicences/handlers/caseload.ts

```diff
@@ -11,29 +14,32 @@ export default class CaseloadRoutes {
     const search = req.query.search as string
     const { user } = res.locals
 
-    const licences = teamView
+    const cases = teamView
       ? await this.caseloadService.getTeamVaryCaseload(user)
       : await this.caseloadService.getStaffVaryCaseload(user)
 
-    const caseloadViewModel = licences
-      .map(licence => {
+    const caseloadViewModel = cases
+      .map(c => {
+        const licence =
+          c.licences.length > 1 ? c.licences.find(l => l.status !== LicenceStatus.ACTIVE) : _.head(c.licences)
+
```

Comment:
Under what circumstances would an individual have a licences list > 1 long? Does the caseload service return all licences for a person together as a list?

Reply:
Is the find !== ACTIVE sufficient to rule out other status codes that might not want to be shown in the vary caseload? e.g. INACTIVE (or APPROVED, and in future, DISCARDED).

Reply:
Someone may have both an ACTIVE and VARIATION_IN_PROGRESS licence at the same time. The vary caseload builder in caseloadService would not return APPROVED or DISCARDED licences.